core/src/main/java/org/elasticsearch/action/get/GetRequest.java

```diff
@@ -101,6 +101,9 @@ public ActionRequestValidationException validate() {
             validationException = ValidateActions.addValidationError("illegal version value [" + version + "] for version type [" + versionType.name() + "]",
                     validationException);
         }
+        if (versionType == VersionType.FORCE) {
```

Comment:
Ahh okay, if the `UpdateHelper` doesn't use this then we should be able to block this with no problem.